### PR TITLE
Created our enumerable

### PR DESCRIPTION
**Pull request by TuyishimireEric**
# Create our enumerable
In this pair programming project, I have collaborated with @a-laarabi and we have done the following in this branch:

- [x] Created a class MyList that has an instance variable @list.
- [x]  Created a module MyEnumerable that implements the following methods (they should have the same functionality as methods in [Enumerable](https://ruby-doc.org/core-3.0.0/Enumerable.html)):
  >  #all?
  >  #any?
  >  #filter

- [x] Created our list and used:
> Test #all?
> Test #any?
> Test #filter

## Cheers and Happy reviewing! :joy_cat:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+Feel free to check the [issues page](https://github.com/TuyishimireEric/Enumerable/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (Reem-lab) [inline README.md]: - [x] I suggest customizing your README file to the milestone for this project so please add the information and the authors info so kindly edit it and send a PR again 📌💥✨ The README is very important to showcase your project 🤗

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi Team, 👋🏻👋🏻

Good job so far! 💥💥💯💪🏻💪🏻
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionality✅
📌 Good commit messages ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (TuyishimireEric) [reply README.md]: # Thank you @Reem-lab for reviewing our project

We had forgotten to customizing our README file to the milestone for this project thank you for your suggestion

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @TuyishimireEric & @a-laarabi ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**